docs/application/dotnet/guides/nlp/overview.md

```diff
@@ -0,0 +1,13 @@
+# Natural Language Processing
+
+The natural language processing features introduce how you can process sentence with Nlp class . They also describe how you can call methods about Nlp in your application.
+
+You can use the following natural language processing features in your .NET applications:
+
+-   [Natural Language Processing](nlp.md)
+
+    You can use natural language processing functionalities in your application, such as language detect, word_tokenize, lemmatize, ne_chunk, and pos_tag. Nlp is a subset of natural language toolkit that specifies an interface and a protocol for basic natural language processin.
```

Comment:
As a suggestion, this sentences can be re-written as:

Nlp is a subset of natural language toolkit that specifies an interface and a protocol for basic natural language processing. Tizen enables you to use Natural Language Process (Nlp) functionalities, such as language detection, parts of speech, word toeknize, entity detection, and lemmatizing. For more information, see the nltk forum.

This is as per my limited knowledge, please correct me if I am wrong.